tools/peptideshaker/peptide_shaker.xml

```diff
@@ -294,7 +295,10 @@
         <section name="exporting_options" expanded="true" title="Exporting options" help="All different outputs PeptideShaker may produce">
 
             <conditional name="zip_conditional">
-                <param name="zip_output_boolean" type="boolean" truevalue="zip" falsevalue="separate" checked="false" label="Compress results into a single zip file" />
+                <param name="zip_output_boolean" type="select" label="Compress results into a single zip file">
```

Comment:
I think the output filters still check for `True` and ``False`. 

Maybe revert and add an exception for the linter (just add the name of the failing linter to a file `.lint_skip` in the tool dir).